extension/src/extension-options/extension-options.html

```diff
@@ -32,6 +32,60 @@
         </div>
       </label>
 
+      <div id="working-days-and-hours">
+        <p>Working Days & Hours</p>
+        <div id="working-days" class="day-selector">
+          <button type="button" data-day-of-week="monday">Mon</button>
```

Comment:
Not sure if there's much of a tangible benefit, but these should _probably_ be styled checkboxes

Reply:
That's a great idea for accessibility.

Reply:
I'm gonna revisit this in the future since we want to get a release out this week! I won't forget, thanks!